src/overlays/actors/ovl_En_Anubice/z_en_anubice.c

```diff
@@ -53,35 +63,441 @@ static ColliderCylinderInit D_809B22D0 = {
     },
     { 29, 103, 0, { 0, 0, 0 } },
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Anubice/func_809B1120.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Anubice/func_809B11C0.s")
+static DamageTable sDamageTable[] = { {
+    0x00, 0xF0, 0xF0, 0xF0, 0xF0, 0xF0, 0xF1, 0xF2, 0xF0, 0xF2, 0xF6, 0x22, 0xF0, 0xF0, 0xF0, 0xF0,
+    0xF0, 0x23, 0x00, 0x00, 0x00, 0x00, 0xF0, 0xF6, 0xF2, 0xF0, 0xFC, 0xF4, 0x00, 0x00, 0x00, 0x00,
+} };
+
+Vec3f D_809B231C = { 0, 0, 0 };
+Vec3f D_809B2328 = { 0, 0, 0 };
+Vec3f D_809B2334 = { 0, 0, 0 };
+
+void EnAnubice_Hover(EnAnubice* this, GlobalContext* globalCtx) {
+    Player* player = PLAYER;
+
+    this->hoverVelocityTimer += 1500.0f;
+    this->targetHeight = player->actor.world.pos.y + this->playerHeightOffset;
+    Math_ApproachF(&this->actor.world.pos.y, this->targetHeight, 0.1f, 10.0f);
+    Math_ApproachF(&this->playerHeightOffset, 10.0f, 0.1f, 0.5f);
+    this->actor.velocity.y = Math_SinS(this->hoverVelocityTimer);
+}
+
+void EnAnubice_SetFireballRot(EnAnubice* this, GlobalContext* globalCtx) {
+    f32 xzdist;
+    f32 x;
+    f32 y;
+    f32 z;
+    Player* player = PLAYER;
+
+    x = player->actor.world.pos.x - this->fireballPos.x;
+    y = player->actor.world.pos.y + 10.0f - this->fireballPos.y;
+    z = player->actor.world.pos.z - this->fireballPos.z;
+    xzdist = sqrtf(SQ(x) + SQ(z));
+
+    this->fireballRot.x = -(s16)(Math_FAtan2F(y, xzdist) * 10430.378f);
+    this->fireballRot.y = (s16)(Math_FAtan2F(x, z) * 10430.378f);
+}
+
+void EnAnubice_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EnAnubice* this = THIS;
+
+    ActorShape_Init(&this->actor.shape, 0.0f, ActorShadow_DrawCircle, 20.0f);
+    SkelAnime_Init(globalCtx, &this->skelAnime, &D_06003990, &D_06000F74, this->jointTable, this->morphTable, 16);
+
+    osSyncPrintf("\n\n");
+    // ☆☆☆☆☆ Anubis occurence ☆☆☆☆☆
+    osSyncPrintf(VT_FGCOL(YELLOW) "☆☆☆☆☆ アヌビス発生 ☆☆☆☆☆ \n" VT_RST);
+
+    this->actor.naviEnemyId = 0x3A;
+
+    Collider_InitCylinder(globalCtx, &this->col);
+    Collider_SetCylinder(globalCtx, &this->col, &this->actor, &sCylinderInit);
+
+    Actor_SetScale(&this->actor, 0.015f);
+
+    this->actor.colChkInfo.damageTable = sDamageTable;
+    this->actor.colChkInfo.mass = 0xFF;
+    this->actor.shape.yOffset = -4230.0f;
+    this->focusHeightOffset = 0.0f;
+    this->actor.flags &= ~1;
+    this->initialPos = this->actor.world.pos;
+    this->actor.targetMode = 3;
+    this->actionFunc = &EnAnubice_FindCurtains;
```

Comment:
```suggestion
    this->actionFunc = EnAnubice_FindCurtains;
```
functions don't need `&`. this applies later, too.